tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectPropertyTest.java

```diff
@@ -135,9 +133,11 @@ public void create () {
 						value = "props of object with id " + nestedProps.get("id", Integer.class);
 					}
 
-					System.out.println("\t" + key + " = " + value);
+					builder.append("\t").append(key).append(" = ").append(value).append("\n");
```

Comment:
This is definitely cleaner than mixing log() and println() calls.

It doesn't matter here at all, but when anyone else reviews this, I'm wondering when the StringBuilder class in libGDX is considered better than StringBuilder in the JDK. The JDK one has a number of intrinsics, if I remember it right, and also has better support from IDE tooling. The libGDX one supports equals() and hashCode().

Also, who keeps coming up with duplicate names? Array, Queue, and StringBuilder are all long-existing names from the JDK. At least StringBuilder actually correctly describes what the class is. `Array<T>` is much closer to an `ArrayList<T>` than a `T[]`, while `Queue` is just as much a stack as it is a queue, and can almost implement `Deque` from the JDK (for double-ended queues). Even AssetManager has some confusing code in AndroidFileHandle that uses Android's AssetManager class and not the libGDX one.

Reply:
StringBuilder collects chars and exposes the array, it has hascode, equals, and a couple utility methods. IIRC the JRE class stores bytes and doesn't expose anything, so you're allocating to get the chars/string out, which you need to do for hashcode or equals. Newer JVMs don't use StringBuilder at all for string concat, though there's a CLI option to enable it.

Names are hard and all the good names are taken! Short names are nice, but less descriptive.